frontend/src/router.ts

```diff
@@ -136,6 +136,22 @@ export const router = new Router({
             }
           ]
         },
+        {
+          path: "te/",
+          component: () => import("@/views/TE/TE.vue"),
```

Comment:
What is the use of this empty component?

Reply:
It nests the routes.

Reply:
I see that it is not TE-specific. Can't we have a common “proxy” component to nest the routes?

Reply:
I have not been able to find a solution like that because not all nested routes have the same routes in them. Also the navigation menu is generated from the routes.